src/sections/Pricing/generateOptions.js

```diff
@@ -19,7 +19,7 @@ function generateOptions(data) {
       monthlyprice: 5,
       yearlyprice: 100,
       byline: "Advanced collaboration for individuals and organizations",
-      byline2:"Everything included in Free, plus...",
+      byline2: "Everything included in Free, plus...",
```

Comment:
I thought that I added this whitespace yesterday. The reason that I mention it is because we’ve been having trouble here with commit timelines being confused.

Reply:
My codebase was 18 commits behind, that might be the reason why your changes aren't reflecting here.